providers/google.go

```diff
@@ -29,6 +29,12 @@ type GoogleProvider struct {
 	GroupValidator func(string) bool
 }
 
+type claims struct {
+	Subject       string `json:"sub"`
+	Email         string `json:"email"`
+	EmailVerified bool   `json:"email_verified"`
```

Comment:
I think `gofmt` needs to be ran on this file.

Reply:
(Sorry for the nitpick!)

Reply:
It didn't produce a diff... maybe I missed something?
What exactly caught your attention?

Reply:
The `type` and json annotation on lines 34 and 35 do not match up:

![image](https://user-images.githubusercontent.com/3449337/57299884-8e543280-70cd-11e9-8f64-2946232352f1.png)

Reply:
Weird, I just tried it in a [Go playground](https://play.golang.org/p/k8wP_jn0M_-) and it still seems off. Is this just my eyes?

Reply:
Huh... that's not what it looks like for me. Maybe something to do with how tabs are rendered. I'll double-check mine.

Reply:
Ok, I just tried it in vim and it looks correct. So I'm going to put it down as a Firefox/font bug on my end - sorry for the noise!